pkg/requests/webhook_endpoints.go

```diff
@@ -40,3 +42,39 @@ func WebhookEndpointsList(ctx context.Context, baseURL, apiVersion, apiKey strin
 
 	return data
 }
+
+// WebhookEndpointCreate creates a new webhook endpoint
+func WebhookEndpointCreate(ctx context.Context, baseURL, apiVersion, apiKey, url, description string, connect bool, profile *config.Profile) error {
+	if strings.TrimSpace(url) == "" {
+		return fmt.Errorf("url cannot be empty")
+	}
+
+	data := []string{
+		"enabled_events[]=*",
```

Comment:
Also one more thing: Is there any way to track what endpoints were created via this gRPC call? Using `enabled_events[]=*` is sort an anti-pattern, but I think it's okay to do it here while the user is developing. But some insights into how often these endpoints are being used could be helpful for us.

Reply:
yes there is telemetry for creating endpoints on the vscode side